installer/bundle_builder/ingredients/deb/download.sh

```diff
@@ -23,11 +23,11 @@ mkdir -p /etc/apt/keyrings
 curl -fsSL https://pkgs.k8s.io/core:/stable:/${KUBERNETES_MAJOR_VERSION}/deb/Release.key |  gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
 
 echo Update apt package index, install kubelet, kubeadm and kubectl
- apt-get update
- chown -Rv _apt:root /bundle/
- chown -R _apt:root /ingredients
- mv cri-containerd-cni-${CONTAINERD_VERSION}-linux-amd64.tar.gz /ingredients/ 
- cd /ingredients 
- apt-get download {kubelet,kubeadm,kubectl}:$ARCH=$KUBERNETES_VERSION
- apt-get download kubernetes-cni
- apt-get download cri-tools:$ARCH=$KUBERNETES_VERSION
+apt-get update
+chown -Rv _apt:root /bundle/
+chown -R _apt:root /ingredients
+mv cri-containerd-cni-${CONTAINERD_VERSION}-linux-amd64.tar.gz /ingredients/ 
+cd /ingredients 
+apt-get download {kubelet,kubeadm,kubectl}:$ARCH=$KUBERNETES_VERSION
```

Comment:
Can we download kubelet, kubeadm and kubectl from k8s repo without relying on apt so that we know exactly which version we are downloading?
Please create a separate Jira to track it. cc: @snslk

Reply:
https://platform9.atlassian.net/browse/KAAP-519